### PR TITLE
Book controller

### PR DESCRIPTION
**Pull request by maxy96**
# Summary
* Created Book controller with routes of index and show
   * Panko Serializer installing 
       * Added to format the book info
   * Wor-paginate installing
       * Added to paginate books on index path
* Routes protected with authentication

# Trello Card
https://trello.com/c/PeTwCjuc/20-2-crear-rutas-index-y-show-para-book
https://trello.com/c/XvwVOyKd/33-1-paginar-los-endpoints-de-listados  

# Screenshot
Captura de Pantalla 2021-06-10 a la(s) 11 13 31
Captura de Pantalla 2021-06-10 a la(s) 11 15 26

### CODE REVIEW

File: app/assets/stylesheets/book.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Book controller here.
```
Comment (blacksam07) [inline app/assets/stylesheets/book.scss]: delete this file

File: app/controllers/api/book_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module Api
+  class BookController < ApplicationController
+    include Wor::Paginate
```
Comment (blacksam07) [inline app/controllers/api/book_controller.rb]: move this to ApplicationController

File: app/helpers/book_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module BookHelper
```
Comment (blacksam07) [inline app/helpers/book_helper.rb]: Useless file

File: config/application.rb [hunk 4]
```diff
@@ -28,6 +28,10 @@ class Application < Rails::Application
         methods: [:get, :post, :options, :delete, :put, :patch]
       end
     end
+
+    Wor::Paginate.configure do |config|
+      config.formatter = Wor::Paginate::Formatters::PankoFormatter
+    end
```
Comment (blacksam07) [inline config/application.rb]: Move this config to wor-paginate initializer 


* Then you can run `rails generate wor:paginate:install` to create the initializer for configuration details, including default formatter class, page and limit params and default page number.

**Review by blacksam07 — CHANGES_REQUESTED** [review]

**Review by blacksam07 — APPROVED** [review]

**Review by redwarewolf — APPROVED** [review]
I'd add rubocop to fix some spacing errors you are having

**Pull request merged**